eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -414,9 +414,10 @@ object Graph {
      * extremes but always bigger than zero so it's guaranteed to never return zero
      */
     def normalize(value: Double, min: Double, max: Double): Double = {
-      if (value <= min) 0.00001D
-      else if (value > max) 0.99999D
-      else (value - min) / (max - min)
+      assert(max > min)
```

Comment:
If you put something here it should be a `require`.

Reply:
Done.